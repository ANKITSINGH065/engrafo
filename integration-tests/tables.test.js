var utils = require("./utils");

test("basic tables render correctly", done => {
  utils.expectBodyToMatchSnapshot("tables/basic.tex", done);
});
