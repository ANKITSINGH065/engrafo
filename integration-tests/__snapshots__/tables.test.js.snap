// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic tables render correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    <dt-article>
      <h1>
        Untitled
      </h1>
      <div
        class="engrafo-metadata"
      >
        <div
          class="authors"
        />
        <div
          class="links"
        />
      </div>
      <figure
        id="table-1"
      >
        <table>
          <caption>
            Table 1: Some numbers.
          </caption>
          <thead>
            <tr
              class="header"
            >
              <th
                style="text-align: center;"
              >
                First column 
              </th>
              <th
                style="text-align: center;"
              >
                Second column 
              </th>
            </tr>
          </thead>
          <tbody>
            <tr
              class="odd"
            >
              <td
                style="text-align: center;"
              >
                123 
              </td>
              <td
                style="text-align: center;"
              >
                456 
              </td>
            </tr>
            <tr
              class="even"
            >
              <td
                style="text-align: center;"
              >
                234 
              </td>
              <td
                style="text-align: center;"
              >
                567 
              </td>
            </tr>
            <tr
              class="odd"
            >
              <td
                style="text-align: center;"
              >
                567 
              </td>
              <td
                style="text-align: center;"
              >
                890 
              </td>
            </tr>
            <tr
              class="even"
            >
              <td
                style="text-align: center;"
              />
            </tr>
          </tbody>
        </table>
      </figure>
      <p>
        Take a look at some cool data in 
        <a
          href="#table-1"
        >
          Table 1
        </a>
        .
      </p>
    </dt-article>
    <dt-appendix>
      <div
        class="l-body"
      />
    </dt-appendix>
    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
